docs/advanced/structure.md

```diff
@@ -226,11 +227,12 @@ process UseMeta {
 }
 
 workflow {
-    Channel.of("Montreal")
-    | map { place -> new Metadata(place) }
-    | map { it + [adapter:"AACGTAGCTTGAC"] }
-    | UseMeta
-    | view
+    place_ch = Channel.of("Montreal")
+        .map { place -> new Metadata(place) }
+        .map { metadata -> metadata + [adapter:"AACGTAGCTTGAC"] }
+
+    UseMeta(place_ch)
+        .view()
 }
 ```
 
```

Comment:
I am not sure what i am doing wrong with the getSampleName function but it is not compiling for me. Might be me